core/ui/public/i18n/it/translation.json

```diff
@@ -995,6 +995,7 @@
         "not_a_valid_ipv4_network": "Indirizzo di rete IPv4 non valido",
         "the_join_code_is_not_correctly_encoded": "Il codice di join non ha il formato corretto",
         "cluster_auth_error": "Risposta non valida",
+        "cluster_connection_error": "Connessione al nodo leader all'indirizzo {value} fallita",
```

Comment:
Translations should be provided from Weblate webapp; I suspect this change might lock tranlastions.